test/fs.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+const path = require('path');
```

Comment:
Please, add an empty line after this.
```suggestion
const path = require('path');

```